File: app/src/androidTest/java/com/melorriaga/pokemon/BaseActivityTest.java
package com.melorriaga.pokemon;

import org.junit.After;
import org.junit.Before;

import static io.appflate.restmock.RESTMockServer.reset;

/**
 * Created by melorriaga on 4/2/17.
 */
public abstract class BaseActivityTest {

    @Before
    public void setUp() throws Exception {
        reset();
    }

    @After
    public void tearDown() throws Exception {
        Thread.sleep(3000);
    }

}

File: app/src/androidTest/kotlin/com/melorriaga/pokemon/BaseActivityTest.kt
package com.melorriaga.pokemon

import io.appflate.restmock.RESTMockServer.reset
import org.junit.After
import org.junit.Before

/**
 * Created by melorriaga on 4/2/17.
 */
abstract class BaseActivityTest {

    @Before
    @Throws(Exception::class)
    fun setUp() {
        reset()
    }

    @After
    @Throws(Exception::class)
    fun tearDown() {
        Thread.sleep(1000)
    }

}
